Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, *args, **kwargs):
 		self.STAGE1B_SIZE         = 0x0001F000
 		self.STAGE2_SIZE          = 0x00020000
 
-		self.EPAYLOAD_SIZE        = 0x00100000
+		self.EPAYLOAD_SIZE        = 0x000FD000
```

Comment:
Any reason to reduce EPAYLOAD_SIZE for QEMU? How about APL?

Reply:
This size is reduced to fit the Slim Bootloader size to 2MB, so that the script for testing firmware update will work.